2025-06-26 11:04:19,994 [INFO] [MainThread] [root/bar_manager.py:61]: Successfully loaded updated config and re-initialised all bars.
2025-06-26 11:04:19,994 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:218]: Stopping worker thread...
2025-06-26 11:04:20,003 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:201]: Exiting thread
2025-06-26 11:04:20,004 [DEBUG] [MainThread] [qasync._windows._IocpProactor/_windows.py:75]: Closing
2025-06-26 11:04:20,039 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 4133848
2025-06-26 11:04:20,039 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 3149642
2025-06-26 11:04:20,211 [INFO] [MainThread] [root/log.py:59]: YASB v1.7.6
2025-06-26 11:04:20,212 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:212]: CLI server started v1.1.2
2025-06-26 11:04:20,212 [INFO] [MainThread] [cli_server/cli_server.py:104]: Log pipe server started
2025-06-26 11:04:20,471 [DEBUG] [MainThread] [tzlocal/win32.py:55]: Looking up time zone info from registry
2025-06-26 11:04:20,604 [DEBUG] [MainThread] [comtypes/__init__.py:149]: CoInitializeEx(None, 2)
2025-06-26 11:04:21,524 [INFO] [MainThread] [root/bar_manager.py:70]: Starting SystemEventListener...
2025-06-26 11:04:21,524 [INFO] [MainThread] [root/bar_manager.py:70]: Starting KomorebiEventListener...
2025-06-26 11:04:21,525 [INFO] [MainThread] [root/watcher.py:54]: Created file watcher for path C:\Users\DOxenbridge\.config\yasb
2025-06-26 11:04:21,534 [INFO] [Dummy-5] [root/event_listener.py:51]: Created named pipe yasb-7e56dcda-5229-11f0-9f38-dc97ba7231fc
2025-06-26 11:04:21,579 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:213]: Starting (proactor: <_IocpProactor overlapped#=0 result#=0>)...
2025-06-26 11:04:21,582 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:192]: Thread started
2025-06-26 11:04:21,583 [DEBUG] [MainThread] [systray_widget/systray.py:530]: Loading state from C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:04:21,595 [DEBUG] [TrayMonitor] [systray_widget/systray.py:61]: Systray thread is starting...
2025-06-26 11:04:21,596 [DEBUG] [TaskbarService] [systray_widget/systray.py:75]: TaskbarService thread is starting...
2025-06-26 11:04:21,642 [INFO] [Dummy-5] [root/event_listener.py:125]: Komorebi connected to named pipe: yasb-7e56dcda-5229-11f0-9f38-dc97ba7231fc
2025-06-26 11:04:21,655 [DEBUG] [TaskbarService] [root/tasks_service.py:75]: Found yasb systray hwnd: 10027232
2025-06-26 11:04:21,655 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 10027232
2025-06-26 11:04:21,687 [DEBUG] [MainThread] [systray_widget/systray.py:278]: Systray thread started
2025-06-26 11:04:21,861 [INFO] [MainThread] [root/weather.py:67]: Fetching new weather data at 2025-06-26 11:04:21.861922
2025-06-26 11:04:21,888 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 10027232
2025-06-26 11:04:21,888 [DEBUG] [TrayMonitor] [systray_widget/tray_monitor.py:208]: Finding real tray hwnd
2025-06-26 11:04:21,897 [DEBUG] [MainThread] [systray_widget/tray_monitor.py:144]: Sending TaskbarCreated message: 49271
2025-06-26 11:04:22,962 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:04:23,959 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:04:23,960 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:04:27,982 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:04:30,971 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:04:30,972 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:04:32,485 [INFO] [MainThread] [root/controller.py:14]: Reloading Application because of config change.
2025-06-26 11:04:32,600 [INFO] [MainThread] [cli_server/cli_server.py:112]: Log pipe server stopped
2025-06-26 11:04:32,600 [DEBUG] [MainThread] [cli_server/cli_server.py:206]: CLI server stopped
2025-06-26 11:04:32,605 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:04:32,605 [DEBUG] [MainThread] [systray_widget/systray.py:506]: Saving state to disk
2025-06-26 11:04:32,605 [DEBUG] [MainThread] [systray_widget/systray.py:512]: Saving state to C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:04:32,607 [INFO] [MainThread] [root/bar_manager.py:61]: Successfully loaded updated config and re-initialised all bars.
2025-06-26 11:04:32,607 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:218]: Stopping worker thread...
2025-06-26 11:04:32,615 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:201]: Exiting thread
2025-06-26 11:04:32,617 [DEBUG] [MainThread] [qasync._windows._IocpProactor/_windows.py:75]: Closing
2025-06-26 11:04:32,662 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 4788032
2025-06-26 11:04:32,663 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 10027232
2025-06-26 11:04:32,923 [INFO] [MainThread] [root/log.py:59]: YASB v1.7.6
2025-06-26 11:04:32,924 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:212]: CLI server started v1.1.2
2025-06-26 11:04:32,924 [INFO] [MainThread] [cli_server/cli_server.py:104]: Log pipe server started
2025-06-26 11:04:33,140 [DEBUG] [MainThread] [tzlocal/win32.py:55]: Looking up time zone info from registry
2025-06-26 11:04:33,287 [DEBUG] [MainThread] [comtypes/__init__.py:149]: CoInitializeEx(None, 2)
2025-06-26 11:04:34,290 [INFO] [MainThread] [root/bar_manager.py:70]: Starting KomorebiEventListener...
2025-06-26 11:04:34,290 [INFO] [MainThread] [root/bar_manager.py:70]: Starting SystemEventListener...
2025-06-26 11:04:34,290 [INFO] [MainThread] [root/watcher.py:54]: Created file watcher for path C:\Users\DOxenbridge\.config\yasb
2025-06-26 11:04:34,309 [INFO] [Dummy-5] [root/event_listener.py:51]: Created named pipe yasb-85f2c7c7-5229-11f0-b7d8-dc97ba7231fc
2025-06-26 11:04:34,344 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:213]: Starting (proactor: <_IocpProactor overlapped#=0 result#=0>)...
2025-06-26 11:04:34,346 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:192]: Thread started
2025-06-26 11:04:34,347 [DEBUG] [MainThread] [systray_widget/systray.py:530]: Loading state from C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:04:34,359 [DEBUG] [TrayMonitor] [systray_widget/systray.py:61]: Systray thread is starting...
2025-06-26 11:04:34,360 [DEBUG] [TaskbarService] [systray_widget/systray.py:75]: TaskbarService thread is starting...
2025-06-26 11:04:34,425 [INFO] [Dummy-5] [root/event_listener.py:125]: Komorebi connected to named pipe: yasb-85f2c7c7-5229-11f0-b7d8-dc97ba7231fc
2025-06-26 11:04:34,441 [DEBUG] [TaskbarService] [root/tasks_service.py:75]: Found yasb systray hwnd: 9374708
2025-06-26 11:04:34,441 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 9374708
2025-06-26 11:04:34,497 [DEBUG] [MainThread] [systray_widget/systray.py:278]: Systray thread started
2025-06-26 11:04:34,675 [INFO] [MainThread] [root/weather.py:67]: Fetching new weather data at 2025-06-26 11:04:34.675188
2025-06-26 11:04:34,697 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 9374708
2025-06-26 11:04:34,698 [DEBUG] [TrayMonitor] [systray_widget/tray_monitor.py:208]: Finding real tray hwnd
2025-06-26 11:04:34,710 [DEBUG] [MainThread] [systray_widget/tray_monitor.py:144]: Sending TaskbarCreated message: 49271
2025-06-26 11:04:35,774 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:04:36,771 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:04:36,772 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:04:40,793 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:04:43,787 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:04:43,788 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:04:44,343 [INFO] [MainThread] [root/controller.py:14]: Reloading Application because of config change.
2025-06-26 11:04:44,455 [INFO] [MainThread] [cli_server/cli_server.py:112]: Log pipe server stopped
2025-06-26 11:04:44,455 [DEBUG] [MainThread] [cli_server/cli_server.py:206]: CLI server stopped
2025-06-26 11:04:44,460 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:04:44,460 [DEBUG] [MainThread] [systray_widget/systray.py:506]: Saving state to disk
2025-06-26 11:04:44,461 [DEBUG] [MainThread] [systray_widget/systray.py:512]: Saving state to C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:04:44,462 [INFO] [MainThread] [root/bar_manager.py:61]: Successfully loaded updated config and re-initialised all bars.
2025-06-26 11:04:44,462 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:218]: Stopping worker thread...
2025-06-26 11:04:44,470 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:201]: Exiting thread
2025-06-26 11:04:44,471 [DEBUG] [MainThread] [qasync._windows._IocpProactor/_windows.py:75]: Closing
2025-06-26 11:04:44,487 [ERROR] [Dummy-5] [root/event_service.py:32]: Failed to emit signal <bound PYQT_SIGNAL k_signal_update of WorkspaceWidget object at 0x0000022B86CA6350>. Removing link to KomorebiEvent.KomorebiUpdate.
2025-06-26 11:04:44,507 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 12654392
2025-06-26 11:04:44,508 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 9374708
2025-06-26 11:04:44,698 [INFO] [MainThread] [root/log.py:59]: YASB v1.7.6
2025-06-26 11:04:44,699 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:212]: CLI server started v1.1.2
2025-06-26 11:04:44,699 [INFO] [MainThread] [cli_server/cli_server.py:104]: Log pipe server started
2025-06-26 11:04:44,951 [DEBUG] [MainThread] [tzlocal/win32.py:55]: Looking up time zone info from registry
2025-06-26 11:04:45,095 [DEBUG] [MainThread] [comtypes/__init__.py:149]: CoInitializeEx(None, 2)
2025-06-26 11:04:45,961 [INFO] [MainThread] [root/bar_manager.py:70]: Starting SystemEventListener...
2025-06-26 11:04:45,961 [INFO] [MainThread] [root/bar_manager.py:70]: Starting KomorebiEventListener...
2025-06-26 11:04:45,961 [INFO] [MainThread] [root/watcher.py:54]: Created file watcher for path C:\Users\DOxenbridge\.config\yasb
2025-06-26 11:04:45,978 [INFO] [Dummy-5] [root/event_listener.py:51]: Created named pipe yasb-8ce7b4f0-5229-11f0-b4a1-dc97ba7231fc
2025-06-26 11:04:45,998 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:213]: Starting (proactor: <_IocpProactor overlapped#=0 result#=0>)...
2025-06-26 11:04:46,000 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:192]: Thread started
2025-06-26 11:04:46,001 [DEBUG] [MainThread] [systray_widget/systray.py:530]: Loading state from C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:04:46,010 [DEBUG] [TrayMonitor] [systray_widget/systray.py:61]: Systray thread is starting...
2025-06-26 11:04:46,010 [DEBUG] [TaskbarService] [systray_widget/systray.py:75]: TaskbarService thread is starting...
2025-06-26 11:04:46,074 [INFO] [Dummy-5] [root/event_listener.py:125]: Komorebi connected to named pipe: yasb-8ce7b4f0-5229-11f0-b4a1-dc97ba7231fc
2025-06-26 11:04:46,089 [DEBUG] [TaskbarService] [root/tasks_service.py:75]: Found yasb systray hwnd: 10620630
2025-06-26 11:04:46,089 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 10620630
2025-06-26 11:04:46,127 [DEBUG] [MainThread] [systray_widget/systray.py:278]: Systray thread started
2025-06-26 11:04:46,262 [INFO] [MainThread] [root/weather.py:67]: Fetching new weather data at 2025-06-26 11:04:46.262958
2025-06-26 11:04:46,328 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 10620630
2025-06-26 11:04:46,329 [DEBUG] [TrayMonitor] [systray_widget/tray_monitor.py:208]: Finding real tray hwnd
2025-06-26 11:04:46,339 [DEBUG] [MainThread] [systray_widget/tray_monitor.py:144]: Sending TaskbarCreated message: 49271
2025-06-26 11:04:47,406 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:04:48,408 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:04:48,409 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:04:52,431 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:04:55,426 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:04:55,427 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:04:57,420 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:05:00,270 [INFO] [MainThread] [root/controller.py:14]: Reloading Application because of config change.
2025-06-26 11:05:00,375 [INFO] [MainThread] [cli_server/cli_server.py:112]: Log pipe server stopped
2025-06-26 11:05:00,375 [DEBUG] [MainThread] [cli_server/cli_server.py:206]: CLI server stopped
2025-06-26 11:05:00,380 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:05:00,381 [DEBUG] [MainThread] [systray_widget/systray.py:506]: Saving state to disk
2025-06-26 11:05:00,381 [DEBUG] [MainThread] [systray_widget/systray.py:512]: Saving state to C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:05:00,382 [INFO] [MainThread] [root/bar_manager.py:61]: Successfully loaded updated config and re-initialised all bars.
2025-06-26 11:05:00,382 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:218]: Stopping worker thread...
2025-06-26 11:05:00,390 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:201]: Exiting thread
2025-06-26 11:05:00,392 [DEBUG] [MainThread] [qasync._windows._IocpProactor/_windows.py:75]: Closing
2025-06-26 11:05:00,413 [ERROR] [Dummy-5] [root/event_service.py:32]: Failed to emit signal <bound PYQT_SIGNAL k_signal_update of WorkspaceWidget object at 0x0000026621016350>. Removing link to KomorebiEvent.KomorebiUpdate.
2025-06-26 11:05:00,428 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 9965310
2025-06-26 11:05:00,428 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 10620630
2025-06-26 11:05:00,428 [DEBUG] [TrayMonitor] [systray_widget/tray_monitor.py:149]: WM_CLOSE received, destroying window 10620630
2025-06-26 11:05:00,616 [INFO] [MainThread] [root/log.py:59]: YASB v1.7.6
2025-06-26 11:05:00,616 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:212]: CLI server started v1.1.2
2025-06-26 11:05:00,617 [INFO] [MainThread] [cli_server/cli_server.py:104]: Log pipe server started
2025-06-26 11:05:00,856 [DEBUG] [MainThread] [tzlocal/win32.py:55]: Looking up time zone info from registry
2025-06-26 11:05:01,004 [DEBUG] [MainThread] [comtypes/__init__.py:149]: CoInitializeEx(None, 2)
2025-06-26 11:05:01,866 [INFO] [MainThread] [root/bar_manager.py:70]: Starting KomorebiEventListener...
2025-06-26 11:05:01,866 [INFO] [MainThread] [root/bar_manager.py:70]: Starting SystemEventListener...
2025-06-26 11:05:01,866 [INFO] [MainThread] [root/watcher.py:54]: Created file watcher for path C:\Users\DOxenbridge\.config\yasb
2025-06-26 11:05:01,883 [INFO] [Dummy-5] [root/event_listener.py:51]: Created named pipe yasb-96629e06-5229-11f0-bf5a-dc97ba7231fc
2025-06-26 11:05:01,909 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:213]: Starting (proactor: <_IocpProactor overlapped#=0 result#=0>)...
2025-06-26 11:05:01,912 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:192]: Thread started
2025-06-26 11:05:01,913 [DEBUG] [MainThread] [systray_widget/systray.py:530]: Loading state from C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:05:01,923 [DEBUG] [TrayMonitor] [systray_widget/systray.py:61]: Systray thread is starting...
2025-06-26 11:05:01,924 [DEBUG] [TaskbarService] [systray_widget/systray.py:75]: TaskbarService thread is starting...
2025-06-26 11:05:01,974 [INFO] [Dummy-5] [root/event_listener.py:125]: Komorebi connected to named pipe: yasb-96629e06-5229-11f0-bf5a-dc97ba7231fc
2025-06-26 11:05:01,985 [DEBUG] [TaskbarService] [root/tasks_service.py:75]: Found yasb systray hwnd: 5180106
2025-06-26 11:05:01,985 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 5180106
2025-06-26 11:05:02,018 [DEBUG] [MainThread] [systray_widget/systray.py:278]: Systray thread started
2025-06-26 11:05:02,192 [INFO] [MainThread] [root/weather.py:67]: Fetching new weather data at 2025-06-26 11:05:02.191339
2025-06-26 11:05:02,219 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 5180106
2025-06-26 11:05:02,220 [DEBUG] [TrayMonitor] [systray_widget/tray_monitor.py:208]: Finding real tray hwnd
2025-06-26 11:05:02,230 [DEBUG] [MainThread] [systray_widget/tray_monitor.py:144]: Sending TaskbarCreated message: 49271
2025-06-26 11:05:03,293 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:05:04,285 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:05:04,286 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:05:08,323 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:05:11,320 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:05:11,320 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:05:12,343 [INFO] [MainThread] [root/controller.py:14]: Reloading Application because of config change.
2025-06-26 11:05:12,456 [INFO] [MainThread] [cli_server/cli_server.py:112]: Log pipe server stopped
2025-06-26 11:05:12,457 [DEBUG] [MainThread] [cli_server/cli_server.py:206]: CLI server stopped
2025-06-26 11:05:12,462 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:05:12,462 [DEBUG] [MainThread] [systray_widget/systray.py:506]: Saving state to disk
2025-06-26 11:05:12,463 [DEBUG] [MainThread] [systray_widget/systray.py:512]: Saving state to C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:05:12,464 [INFO] [MainThread] [root/bar_manager.py:61]: Successfully loaded updated config and re-initialised all bars.
2025-06-26 11:05:12,464 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:218]: Stopping worker thread...
2025-06-26 11:05:12,472 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:201]: Exiting thread
2025-06-26 11:05:12,473 [DEBUG] [MainThread] [qasync._windows._IocpProactor/_windows.py:75]: Closing
2025-06-26 11:05:12,508 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 1179764
2025-06-26 11:05:12,508 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 5180106
2025-06-26 11:05:12,701 [INFO] [MainThread] [root/log.py:59]: YASB v1.7.6
2025-06-26 11:05:12,701 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:212]: CLI server started v1.1.2
2025-06-26 11:05:12,702 [INFO] [MainThread] [cli_server/cli_server.py:104]: Log pipe server started
2025-06-26 11:05:12,950 [DEBUG] [MainThread] [tzlocal/win32.py:55]: Looking up time zone info from registry
2025-06-26 11:05:13,103 [DEBUG] [MainThread] [comtypes/__init__.py:149]: CoInitializeEx(None, 2)
2025-06-26 11:05:13,995 [INFO] [MainThread] [root/bar_manager.py:70]: Starting KomorebiEventListener...
2025-06-26 11:05:13,995 [INFO] [MainThread] [root/bar_manager.py:70]: Starting SystemEventListener...
2025-06-26 11:05:13,996 [INFO] [MainThread] [root/watcher.py:54]: Created file watcher for path C:\Users\DOxenbridge\.config\yasb
2025-06-26 11:05:14,010 [INFO] [Dummy-5] [root/event_listener.py:51]: Created named pipe yasb-9d9d4129-5229-11f0-8674-dc97ba7231fc
2025-06-26 11:05:14,039 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:213]: Starting (proactor: <_IocpProactor overlapped#=0 result#=0>)...
2025-06-26 11:05:14,040 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:192]: Thread started
2025-06-26 11:05:14,041 [DEBUG] [MainThread] [systray_widget/systray.py:530]: Loading state from C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:05:14,051 [DEBUG] [TaskbarService] [systray_widget/systray.py:75]: TaskbarService thread is starting...
2025-06-26 11:05:14,053 [DEBUG] [TrayMonitor] [systray_widget/systray.py:61]: Systray thread is starting...
2025-06-26 11:05:14,109 [INFO] [Dummy-5] [root/event_listener.py:125]: Komorebi connected to named pipe: yasb-9d9d4129-5229-11f0-8674-dc97ba7231fc
2025-06-26 11:05:14,124 [DEBUG] [TaskbarService] [root/tasks_service.py:75]: Found yasb systray hwnd: 24445036
2025-06-26 11:05:14,124 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 24445036
2025-06-26 11:05:14,196 [DEBUG] [MainThread] [systray_widget/systray.py:278]: Systray thread started
2025-06-26 11:05:14,313 [INFO] [MainThread] [root/weather.py:67]: Fetching new weather data at 2025-06-26 11:05:14.313564
2025-06-26 11:05:14,398 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 24445036
2025-06-26 11:05:14,398 [DEBUG] [TrayMonitor] [systray_widget/tray_monitor.py:208]: Finding real tray hwnd
2025-06-26 11:05:14,408 [DEBUG] [MainThread] [systray_widget/tray_monitor.py:144]: Sending TaskbarCreated message: 49271
2025-06-26 11:05:15,471 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:05:16,471 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:05:16,471 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:05:20,487 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:05:23,516 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:05:23,516 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:05:25,479 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:05:30,479 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:05:35,496 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:05:36,853 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:05:36,853 [DEBUG] [MainThread] [systray_widget/systray.py:506]: Saving state to disk
2025-06-26 11:05:36,854 [DEBUG] [MainThread] [systray_widget/systray.py:512]: Saving state to C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:05:40,494 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:05:45,483 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:05:50,479 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:05:55,486 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:06:00,480 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:06:05,484 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:06:10,492 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:06:15,481 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:06:17,719 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:06:20,489 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:06:25,500 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:06:27,905 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:06:30,490 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 10916. Err: 5
2025-06-26 11:06:44,162 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:06:45,166 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:06:45,168 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:06:49,177 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:06:54,170 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:06:59,170 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:07:09,175 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:07:11,930 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:07:14,174 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:07:19,170 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:07:24,166 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:07:29,175 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:07:34,180 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:07:39,191 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:07:44,183 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:07:44,418 [INFO] [MainThread] [root/controller.py:14]: Reloading Application because of config change.
2025-06-26 11:07:44,533 [INFO] [MainThread] [cli_server/cli_server.py:112]: Log pipe server stopped
2025-06-26 11:07:44,533 [DEBUG] [MainThread] [cli_server/cli_server.py:206]: CLI server stopped
2025-06-26 11:07:44,538 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:07:44,538 [DEBUG] [MainThread] [systray_widget/systray.py:506]: Saving state to disk
2025-06-26 11:07:44,538 [DEBUG] [MainThread] [systray_widget/systray.py:512]: Saving state to C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:07:44,539 [INFO] [MainThread] [root/bar_manager.py:61]: Successfully loaded updated config and re-initialised all bars.
2025-06-26 11:07:44,540 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:218]: Stopping worker thread...
2025-06-26 11:07:44,549 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:201]: Exiting thread
2025-06-26 11:07:44,550 [DEBUG] [MainThread] [qasync._windows._IocpProactor/_windows.py:75]: Closing
2025-06-26 11:07:44,583 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 5836736
2025-06-26 11:07:44,583 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 24445036
2025-06-26 11:07:44,834 [INFO] [MainThread] [root/log.py:59]: YASB v1.7.6
2025-06-26 11:07:44,835 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:212]: CLI server started v1.1.2
2025-06-26 11:07:44,835 [INFO] [MainThread] [cli_server/cli_server.py:104]: Log pipe server started
2025-06-26 11:07:45,169 [DEBUG] [MainThread] [tzlocal/win32.py:55]: Looking up time zone info from registry
2025-06-26 11:07:45,311 [DEBUG] [MainThread] [comtypes/__init__.py:149]: CoInitializeEx(None, 2)
2025-06-26 11:07:46,360 [INFO] [MainThread] [root/bar_manager.py:70]: Starting SystemEventListener...
2025-06-26 11:07:46,360 [INFO] [MainThread] [root/bar_manager.py:70]: Starting KomorebiEventListener...
2025-06-26 11:07:46,361 [INFO] [MainThread] [root/watcher.py:54]: Created file watcher for path C:\Users\DOxenbridge\.config\yasb
2025-06-26 11:07:46,379 [INFO] [Dummy-5] [root/event_listener.py:51]: Created named pipe yasb-f86e5e43-5229-11f0-8a47-dc97ba7231fc
2025-06-26 11:07:46,398 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:213]: Starting (proactor: <_IocpProactor overlapped#=0 result#=0>)...
2025-06-26 11:07:46,400 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:192]: Thread started
2025-06-26 11:07:46,400 [DEBUG] [MainThread] [systray_widget/systray.py:530]: Loading state from C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:07:46,410 [DEBUG] [TrayMonitor] [systray_widget/systray.py:61]: Systray thread is starting...
2025-06-26 11:07:46,412 [DEBUG] [TaskbarService] [systray_widget/systray.py:75]: TaskbarService thread is starting...
2025-06-26 11:07:46,485 [INFO] [Dummy-5] [root/event_listener.py:125]: Komorebi connected to named pipe: yasb-f86e5e43-5229-11f0-8a47-dc97ba7231fc
2025-06-26 11:07:46,489 [DEBUG] [TaskbarService] [root/tasks_service.py:75]: Found yasb systray hwnd: 3081552
2025-06-26 11:07:46,489 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 3081552
2025-06-26 11:07:46,528 [DEBUG] [MainThread] [systray_widget/systray.py:278]: Systray thread started
2025-06-26 11:07:46,729 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 3081552
2025-06-26 11:07:46,730 [DEBUG] [TrayMonitor] [systray_widget/tray_monitor.py:208]: Finding real tray hwnd
2025-06-26 11:07:46,741 [DEBUG] [MainThread] [systray_widget/tray_monitor.py:144]: Sending TaskbarCreated message: 49271
2025-06-26 11:07:46,862 [INFO] [MainThread] [root/weather.py:67]: Fetching new weather data at 2025-06-26 11:07:46.862855
2025-06-26 11:07:47,783 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:07:48,787 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:07:48,787 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:07:52,794 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:07:55,822 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:07:55,822 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:07:57,795 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:08:00,877 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:08:00,877 [DEBUG] [MainThread] [systray_widget/systray.py:506]: Saving state to disk
2025-06-26 11:08:00,878 [DEBUG] [MainThread] [systray_widget/systray.py:512]: Saving state to C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:08:02,787 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:08:07,801 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:08:12,788 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:08:17,789 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:08:22,461 [INFO] [MainThread] [root/controller.py:14]: Reloading Application because of config change.
2025-06-26 11:08:22,567 [INFO] [MainThread] [cli_server/cli_server.py:112]: Log pipe server stopped
2025-06-26 11:08:22,567 [DEBUG] [MainThread] [cli_server/cli_server.py:206]: CLI server stopped
2025-06-26 11:08:22,570 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:08:22,570 [DEBUG] [MainThread] [systray_widget/systray.py:506]: Saving state to disk
2025-06-26 11:08:22,570 [DEBUG] [MainThread] [systray_widget/systray.py:512]: Saving state to C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:08:22,571 [INFO] [MainThread] [root/bar_manager.py:61]: Successfully loaded updated config and re-initialised all bars.
2025-06-26 11:08:22,571 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:218]: Stopping worker thread...
2025-06-26 11:08:22,582 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:201]: Exiting thread
2025-06-26 11:08:22,585 [DEBUG] [MainThread] [qasync._windows._IocpProactor/_windows.py:75]: Closing
2025-06-26 11:08:22,624 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 5115712
2025-06-26 11:08:22,624 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 3081552
2025-06-26 11:08:22,800 [INFO] [MainThread] [root/log.py:59]: YASB v1.7.6
2025-06-26 11:08:22,801 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:212]: CLI server started v1.1.2
2025-06-26 11:08:22,801 [INFO] [MainThread] [cli_server/cli_server.py:104]: Log pipe server started
2025-06-26 11:08:23,082 [DEBUG] [MainThread] [tzlocal/win32.py:55]: Looking up time zone info from registry
2025-06-26 11:08:23,221 [DEBUG] [MainThread] [comtypes/__init__.py:149]: CoInitializeEx(None, 2)
2025-06-26 11:08:24,111 [INFO] [MainThread] [root/bar_manager.py:70]: Starting KomorebiEventListener...
2025-06-26 11:08:24,111 [INFO] [MainThread] [root/bar_manager.py:70]: Starting SystemEventListener...
2025-06-26 11:08:24,111 [INFO] [MainThread] [root/watcher.py:54]: Created file watcher for path C:\Users\DOxenbridge\.config\yasb
2025-06-26 11:08:24,128 [INFO] [Dummy-5] [root/event_listener.py:51]: Created named pipe yasb-0eeeac81-522a-11f0-862d-dc97ba7231fc
2025-06-26 11:08:24,148 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:213]: Starting (proactor: <_IocpProactor overlapped#=0 result#=0>)...
2025-06-26 11:08:24,149 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:192]: Thread started
2025-06-26 11:08:24,150 [DEBUG] [MainThread] [systray_widget/systray.py:530]: Loading state from C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:08:24,160 [DEBUG] [TrayMonitor] [systray_widget/systray.py:61]: Systray thread is starting...
2025-06-26 11:08:24,160 [DEBUG] [TaskbarService] [systray_widget/systray.py:75]: TaskbarService thread is starting...
2025-06-26 11:08:24,220 [INFO] [Dummy-5] [root/event_listener.py:125]: Komorebi connected to named pipe: yasb-0eeeac81-522a-11f0-862d-dc97ba7231fc
2025-06-26 11:08:24,220 [DEBUG] [TaskbarService] [root/tasks_service.py:75]: Found yasb systray hwnd: 7409500
2025-06-26 11:08:24,220 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 7409500
2025-06-26 11:08:24,259 [DEBUG] [MainThread] [systray_widget/systray.py:278]: Systray thread started
2025-06-26 11:08:24,431 [INFO] [MainThread] [root/weather.py:67]: Fetching new weather data at 2025-06-26 11:08:24.431437
2025-06-26 11:08:24,460 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 7409500
2025-06-26 11:08:24,460 [DEBUG] [TrayMonitor] [systray_widget/tray_monitor.py:208]: Finding real tray hwnd
2025-06-26 11:08:24,471 [DEBUG] [MainThread] [systray_widget/tray_monitor.py:144]: Sending TaskbarCreated message: 49271
2025-06-26 11:08:25,508 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:08:26,509 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:08:26,509 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:08:30,532 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:08:33,564 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:08:33,565 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:08:35,523 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:08:40,527 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:08:42,301 [INFO] [MainThread] [root/controller.py:14]: Reloading Application because of config change.
2025-06-26 11:08:42,403 [INFO] [MainThread] [cli_server/cli_server.py:112]: Log pipe server stopped
2025-06-26 11:08:42,403 [DEBUG] [MainThread] [cli_server/cli_server.py:206]: CLI server stopped
2025-06-26 11:08:42,406 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:08:42,406 [DEBUG] [MainThread] [systray_widget/systray.py:506]: Saving state to disk
2025-06-26 11:08:42,406 [DEBUG] [MainThread] [systray_widget/systray.py:512]: Saving state to C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:08:42,407 [INFO] [MainThread] [root/bar_manager.py:61]: Successfully loaded updated config and re-initialised all bars.
2025-06-26 11:08:42,407 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:218]: Stopping worker thread...
2025-06-26 11:08:42,419 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:201]: Exiting thread
2025-06-26 11:08:42,421 [DEBUG] [MainThread] [qasync._windows._IocpProactor/_windows.py:75]: Closing
2025-06-26 11:08:42,460 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 10491492
2025-06-26 11:08:42,460 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 7409500
2025-06-26 11:08:42,635 [INFO] [MainThread] [root/log.py:59]: YASB v1.7.6
2025-06-26 11:08:42,636 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:212]: CLI server started v1.1.2
2025-06-26 11:08:42,636 [INFO] [MainThread] [cli_server/cli_server.py:104]: Log pipe server started
2025-06-26 11:08:42,897 [DEBUG] [MainThread] [tzlocal/win32.py:55]: Looking up time zone info from registry
2025-06-26 11:08:43,035 [DEBUG] [MainThread] [comtypes/__init__.py:149]: CoInitializeEx(None, 2)
2025-06-26 11:08:44,077 [INFO] [MainThread] [root/bar_manager.py:70]: Starting KomorebiEventListener...
2025-06-26 11:08:44,077 [INFO] [MainThread] [root/bar_manager.py:70]: Starting SystemEventListener...
2025-06-26 11:08:44,078 [INFO] [MainThread] [root/watcher.py:54]: Created file watcher for path C:\Users\DOxenbridge\.config\yasb
2025-06-26 11:08:44,078 [INFO] [Dummy-5] [root/event_listener.py:51]: Created named pipe yasb-1ad54012-522a-11f0-9a52-dc97ba7231fc
2025-06-26 11:08:44,125 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:213]: Starting (proactor: <_IocpProactor overlapped#=0 result#=0>)...
2025-06-26 11:08:44,128 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:192]: Thread started
2025-06-26 11:08:44,128 [DEBUG] [MainThread] [systray_widget/systray.py:530]: Loading state from C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:08:44,138 [DEBUG] [TaskbarService] [systray_widget/systray.py:75]: TaskbarService thread is starting...
2025-06-26 11:08:44,138 [DEBUG] [TrayMonitor] [systray_widget/systray.py:61]: Systray thread is starting...
2025-06-26 11:08:44,201 [INFO] [Dummy-5] [root/event_listener.py:125]: Komorebi connected to named pipe: yasb-1ad54012-522a-11f0-9a52-dc97ba7231fc
2025-06-26 11:08:44,203 [DEBUG] [TaskbarService] [root/tasks_service.py:75]: Found yasb systray hwnd: 9898996
2025-06-26 11:08:44,203 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 9898996
2025-06-26 11:08:44,244 [DEBUG] [MainThread] [systray_widget/systray.py:278]: Systray thread started
2025-06-26 11:08:44,446 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 9898996
2025-06-26 11:08:44,447 [DEBUG] [TrayMonitor] [systray_widget/tray_monitor.py:208]: Finding real tray hwnd
2025-06-26 11:08:44,455 [DEBUG] [MainThread] [systray_widget/tray_monitor.py:144]: Sending TaskbarCreated message: 49271
2025-06-26 11:08:44,535 [INFO] [MainThread] [root/weather.py:67]: Fetching new weather data at 2025-06-26 11:08:44.535218
2025-06-26 11:08:45,505 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:08:46,507 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:08:46,508 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:08:50,533 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:08:53,548 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:08:53,549 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:08:55,539 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:09:00,547 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:09:05,546 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:09:10,540 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:09:15,551 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:09:20,547 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:09:25,548 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:09:30,555 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:09:35,543 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:09:40,542 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:09:45,552 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:09:50,549 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:09:55,246 [INFO] [MainThread] [root/controller.py:14]: Reloading Application because of config change.
2025-06-26 11:09:55,353 [INFO] [MainThread] [cli_server/cli_server.py:112]: Log pipe server stopped
2025-06-26 11:09:55,353 [DEBUG] [MainThread] [cli_server/cli_server.py:206]: CLI server stopped
2025-06-26 11:09:55,357 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:09:55,357 [DEBUG] [MainThread] [systray_widget/systray.py:506]: Saving state to disk
2025-06-26 11:09:55,358 [DEBUG] [MainThread] [systray_widget/systray.py:512]: Saving state to C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:09:55,358 [INFO] [MainThread] [root/bar_manager.py:61]: Successfully loaded updated config and re-initialised all bars.
2025-06-26 11:09:55,358 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:218]: Stopping worker thread...
2025-06-26 11:09:55,368 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:201]: Exiting thread
2025-06-26 11:09:55,369 [DEBUG] [MainThread] [qasync._windows._IocpProactor/_windows.py:75]: Closing
2025-06-26 11:09:55,404 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 5117052
2025-06-26 11:09:55,404 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 9898996
2025-06-26 11:09:55,650 [INFO] [MainThread] [root/log.py:59]: YASB v1.7.6
2025-06-26 11:09:55,651 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:212]: CLI server started v1.1.2
2025-06-26 11:09:55,651 [INFO] [MainThread] [cli_server/cli_server.py:104]: Log pipe server started
2025-06-26 11:09:55,921 [DEBUG] [MainThread] [tzlocal/win32.py:55]: Looking up time zone info from registry
2025-06-26 11:09:56,083 [DEBUG] [MainThread] [comtypes/__init__.py:149]: CoInitializeEx(None, 2)
2025-06-26 11:09:57,010 [INFO] [MainThread] [root/bar_manager.py:70]: Starting KomorebiEventListener...
2025-06-26 11:09:57,010 [INFO] [MainThread] [root/bar_manager.py:70]: Starting SystemEventListener...
2025-06-26 11:09:57,011 [INFO] [MainThread] [root/watcher.py:54]: Created file watcher for path C:\Users\DOxenbridge\.config\yasb
2025-06-26 11:09:57,017 [INFO] [Dummy-5] [root/event_listener.py:51]: Created named pipe yasb-464dea5c-522a-11f0-90dc-dc97ba7231fc
2025-06-26 11:09:57,056 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:213]: Starting (proactor: <_IocpProactor overlapped#=0 result#=0>)...
2025-06-26 11:09:57,057 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:192]: Thread started
2025-06-26 11:09:57,057 [DEBUG] [MainThread] [systray_widget/systray.py:530]: Loading state from C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:09:57,067 [DEBUG] [TrayMonitor] [systray_widget/systray.py:61]: Systray thread is starting...
2025-06-26 11:09:57,068 [DEBUG] [TaskbarService] [systray_widget/systray.py:75]: TaskbarService thread is starting...
2025-06-26 11:09:57,136 [DEBUG] [TaskbarService] [root/tasks_service.py:75]: Found yasb systray hwnd: 5507786
2025-06-26 11:09:57,136 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 5507786
2025-06-26 11:09:57,139 [INFO] [Dummy-5] [root/event_listener.py:125]: Komorebi connected to named pipe: yasb-464dea5c-522a-11f0-90dc-dc97ba7231fc
2025-06-26 11:09:57,178 [DEBUG] [MainThread] [systray_widget/systray.py:278]: Systray thread started
2025-06-26 11:09:57,345 [INFO] [MainThread] [root/weather.py:67]: Fetching new weather data at 2025-06-26 11:09:57.345899
2025-06-26 11:09:57,380 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 5507786
2025-06-26 11:09:57,380 [DEBUG] [TrayMonitor] [systray_widget/tray_monitor.py:208]: Finding real tray hwnd
2025-06-26 11:09:57,392 [DEBUG] [MainThread] [systray_widget/tray_monitor.py:144]: Sending TaskbarCreated message: 49271
2025-06-26 11:09:58,433 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:09:59,429 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:09:59,430 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:10:03,445 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:10:06,330 [INFO] [MainThread] [root/controller.py:14]: Reloading Application because of config change.
2025-06-26 11:10:06,437 [INFO] [MainThread] [cli_server/cli_server.py:112]: Log pipe server stopped
2025-06-26 11:10:06,437 [DEBUG] [MainThread] [cli_server/cli_server.py:206]: CLI server stopped
2025-06-26 11:10:06,441 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:10:06,442 [DEBUG] [MainThread] [systray_widget/systray.py:506]: Saving state to disk
2025-06-26 11:10:06,442 [DEBUG] [MainThread] [systray_widget/systray.py:512]: Saving state to C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:10:06,443 [INFO] [MainThread] [root/bar_manager.py:61]: Successfully loaded updated config and re-initialised all bars.
2025-06-26 11:10:06,444 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:218]: Stopping worker thread...
2025-06-26 11:10:06,453 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:201]: Exiting thread
2025-06-26 11:10:06,454 [DEBUG] [MainThread] [qasync._windows._IocpProactor/_windows.py:75]: Closing
2025-06-26 11:10:06,490 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 6427278
2025-06-26 11:10:06,491 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 5507786
2025-06-26 11:10:06,666 [INFO] [MainThread] [root/log.py:59]: YASB v1.7.6
2025-06-26 11:10:06,667 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:212]: CLI server started v1.1.2
2025-06-26 11:10:06,667 [INFO] [MainThread] [cli_server/cli_server.py:104]: Log pipe server started
2025-06-26 11:10:06,937 [DEBUG] [MainThread] [tzlocal/win32.py:55]: Looking up time zone info from registry
2025-06-26 11:10:07,074 [DEBUG] [MainThread] [comtypes/__init__.py:149]: CoInitializeEx(None, 2)
2025-06-26 11:10:07,909 [INFO] [MainThread] [root/bar_manager.py:70]: Starting KomorebiEventListener...
2025-06-26 11:10:07,909 [INFO] [MainThread] [root/bar_manager.py:70]: Starting SystemEventListener...
2025-06-26 11:10:07,909 [INFO] [MainThread] [root/watcher.py:54]: Created file watcher for path C:\Users\DOxenbridge\.config\yasb
2025-06-26 11:10:07,923 [INFO] [Dummy-5] [root/event_listener.py:51]: Created named pipe yasb-4ccd052a-522a-11f0-bcd6-dc97ba7231fc
2025-06-26 11:10:07,952 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:213]: Starting (proactor: <_IocpProactor overlapped#=0 result#=0>)...
2025-06-26 11:10:07,955 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:192]: Thread started
2025-06-26 11:10:07,956 [DEBUG] [MainThread] [systray_widget/systray.py:530]: Loading state from C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:10:07,964 [DEBUG] [TrayMonitor] [systray_widget/systray.py:61]: Systray thread is starting...
2025-06-26 11:10:07,965 [DEBUG] [TaskbarService] [systray_widget/systray.py:75]: TaskbarService thread is starting...
2025-06-26 11:10:08,016 [INFO] [Dummy-5] [root/event_listener.py:125]: Komorebi connected to named pipe: yasb-4ccd052a-522a-11f0-bcd6-dc97ba7231fc
2025-06-26 11:10:08,024 [DEBUG] [TaskbarService] [root/tasks_service.py:75]: Found yasb systray hwnd: 16193092
2025-06-26 11:10:08,024 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 16193092
2025-06-26 11:10:08,053 [DEBUG] [MainThread] [systray_widget/systray.py:278]: Systray thread started
2025-06-26 11:10:08,220 [INFO] [MainThread] [root/weather.py:67]: Fetching new weather data at 2025-06-26 11:10:08.220488
2025-06-26 11:10:08,228 [DEBUG] [TrayMonitor] [systray_widget/tray_monitor.py:208]: Finding real tray hwnd
2025-06-26 11:10:08,255 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 16193092
2025-06-26 11:10:08,269 [DEBUG] [MainThread] [systray_widget/tray_monitor.py:144]: Sending TaskbarCreated message: 49271
2025-06-26 11:10:09,307 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:10:10,311 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:10:10,312 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:10:14,330 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:10:17,331 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:10:17,332 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:10:19,325 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:10:24,332 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:10:29,334 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:10:34,322 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:10:39,334 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:10:44,323 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:10:49,323 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:10:54,338 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:10:59,329 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:11:04,332 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:11:09,327 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:11:14,334 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:11:19,340 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:11:24,335 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:11:29,326 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:11:34,332 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:11:39,327 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:11:44,326 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:11:49,337 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:11:54,325 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:11:59,335 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:12:04,337 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:12:09,332 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:12:14,334 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:12:19,334 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:12:24,332 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:12:27,017 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:12:27,017 [DEBUG] [MainThread] [systray_widget/systray.py:506]: Saving state to disk
2025-06-26 11:12:27,017 [DEBUG] [MainThread] [systray_widget/systray.py:512]: Saving state to C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:12:28,391 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:12:28,391 [DEBUG] [MainThread] [systray_widget/systray.py:506]: Saving state to disk
2025-06-26 11:12:28,392 [DEBUG] [MainThread] [systray_widget/systray.py:512]: Saving state to C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:12:29,336 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:12:30,230 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:12:30,230 [DEBUG] [MainThread] [systray_widget/systray.py:506]: Saving state to disk
2025-06-26 11:12:30,230 [DEBUG] [MainThread] [systray_widget/systray.py:512]: Saving state to C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:12:34,340 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:12:39,333 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:12:44,326 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:12:49,333 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:12:54,323 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:12:59,329 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:13:04,328 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:13:09,324 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:13:14,331 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:13:19,326 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:13:24,336 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:13:29,337 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:13:34,325 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:13:39,327 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:13:44,330 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:13:49,337 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:13:54,328 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:13:59,323 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:14:04,341 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:14:09,326 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:14:14,337 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:14:19,332 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:14:24,336 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:14:29,338 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:14:34,333 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:14:39,329 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:14:44,333 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:14:49,336 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:14:54,331 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:14:59,326 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:15:04,329 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:15:09,327 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:15:14,334 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:15:19,322 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:15:24,328 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:15:29,331 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:15:34,338 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:15:39,335 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:15:44,328 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:15:49,333 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:15:54,343 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:15:59,332 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:16:04,330 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:16:09,335 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:16:14,333 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:16:19,336 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:16:24,330 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:16:29,335 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:16:34,333 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:16:39,335 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:16:44,331 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:16:49,325 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:16:54,328 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:16:59,331 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:17:04,339 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:17:09,332 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:17:14,324 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:17:19,331 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:17:24,337 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:17:29,328 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:17:34,330 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:17:39,338 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:17:44,333 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:17:49,332 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:17:54,334 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:17:59,331 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:18:04,328 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:18:09,332 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:18:14,326 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:18:19,337 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:18:24,340 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:18:29,336 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:18:34,328 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:18:39,330 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:18:44,340 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:18:44,847 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:253]: CLI server received command: toggle-bar
2025-06-26 11:18:49,327 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:18:54,328 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:18:55,363 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:253]: CLI server received command: toggle-bar
2025-06-26 11:18:56,755 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:253]: CLI server received command: toggle-bar
2025-06-26 11:18:59,326 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:19:00,856 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:253]: CLI server received command: stop
2025-06-26 11:19:00,856 [INFO] [Thread-1 (_run_server)] [root/controller.py:29]: Exiting Application from CLI...
2025-06-26 11:19:00,971 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:112]: Log pipe server stopped
2025-06-26 11:19:00,971 [DEBUG] [Thread-1 (_run_server)] [cli_server/cli_server.py:206]: CLI server stopped
2025-06-26 11:19:00,972 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:218]: Stopping worker thread...
2025-06-26 11:19:00,986 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:201]: Exiting thread
2025-06-26 11:19:00,988 [DEBUG] [MainThread] [qasync._windows._IocpProactor/_windows.py:75]: Closing
2025-06-26 11:19:01,015 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 13176780
2025-06-26 11:19:01,015 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 16193092
2025-06-26 11:19:04,700 [INFO] [MainThread] [root/log.py:59]: YASB v1.7.6
2025-06-26 11:19:04,701 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:212]: CLI server started v1.1.2
2025-06-26 11:19:04,701 [INFO] [MainThread] [cli_server/cli_server.py:104]: Log pipe server started
2025-06-26 11:19:04,831 [DEBUG] [MainThread] [tzlocal/win32.py:55]: Looking up time zone info from registry
2025-06-26 11:19:04,983 [DEBUG] [MainThread] [comtypes/__init__.py:149]: CoInitializeEx(None, 2)
2025-06-26 11:19:05,982 [INFO] [MainThread] [root/bar_manager.py:70]: Starting SystemEventListener...
2025-06-26 11:19:05,983 [INFO] [MainThread] [root/bar_manager.py:70]: Starting KomorebiEventListener...
2025-06-26 11:19:05,984 [INFO] [MainThread] [root/watcher.py:54]: Created file watcher for path C:\Users\DOxenbridge\.config\yasb
2025-06-26 11:19:06,001 [INFO] [Dummy-5] [root/event_listener.py:51]: Created named pipe yasb-8d84adf3-522b-11f0-a0ab-dc97ba7231fc
2025-06-26 11:19:06,028 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:213]: Starting (proactor: <_IocpProactor overlapped#=0 result#=0>)...
2025-06-26 11:19:06,030 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:192]: Thread started
2025-06-26 11:19:06,031 [DEBUG] [MainThread] [systray_widget/systray.py:530]: Loading state from C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:19:06,042 [DEBUG] [TrayMonitor] [systray_widget/systray.py:61]: Systray thread is starting...
2025-06-26 11:19:06,043 [DEBUG] [TaskbarService] [systray_widget/systray.py:75]: TaskbarService thread is starting...
2025-06-26 11:19:06,106 [INFO] [Dummy-5] [root/event_listener.py:125]: Komorebi connected to named pipe: yasb-8d84adf3-522b-11f0-a0ab-dc97ba7231fc
2025-06-26 11:19:06,115 [DEBUG] [TaskbarService] [root/tasks_service.py:75]: Found yasb systray hwnd: 10686206
2025-06-26 11:19:06,115 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 10686206
2025-06-26 11:19:06,159 [DEBUG] [MainThread] [systray_widget/systray.py:278]: Systray thread started
2025-06-26 11:19:06,360 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 10686206
2025-06-26 11:19:06,361 [DEBUG] [TrayMonitor] [systray_widget/tray_monitor.py:208]: Finding real tray hwnd
2025-06-26 11:19:06,374 [DEBUG] [MainThread] [systray_widget/tray_monitor.py:144]: Sending TaskbarCreated message: 49271
2025-06-26 11:19:06,494 [INFO] [MainThread] [root/weather.py:67]: Fetching new weather data at 2025-06-26 11:19:06.494399
2025-06-26 11:19:07,437 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:19:08,432 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:19:08,432 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:19:12,452 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:19:15,489 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:19:15,489 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:19:17,455 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:19:22,459 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:19:27,465 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:19:32,473 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:19:37,467 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:19:42,481 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:19:47,479 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:19:52,472 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:19:57,050 [INFO] [MainThread] [root/controller.py:14]: Reloading Application because of config change.
2025-06-26 11:19:57,162 [INFO] [MainThread] [cli_server/cli_server.py:112]: Log pipe server stopped
2025-06-26 11:19:57,163 [DEBUG] [MainThread] [cli_server/cli_server.py:206]: CLI server stopped
2025-06-26 11:19:57,168 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:19:57,168 [DEBUG] [MainThread] [systray_widget/systray.py:506]: Saving state to disk
2025-06-26 11:19:57,168 [DEBUG] [MainThread] [systray_widget/systray.py:512]: Saving state to C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:19:57,169 [INFO] [MainThread] [root/bar_manager.py:61]: Successfully loaded updated config and re-initialised all bars.
2025-06-26 11:19:57,169 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:218]: Stopping worker thread...
2025-06-26 11:19:57,178 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:201]: Exiting thread
2025-06-26 11:19:57,180 [DEBUG] [MainThread] [qasync._windows._IocpProactor/_windows.py:75]: Closing
2025-06-26 11:19:57,215 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 7671644
2025-06-26 11:19:57,215 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 10686206
2025-06-26 11:19:57,402 [INFO] [MainThread] [root/log.py:59]: YASB v1.7.6
2025-06-26 11:19:57,403 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:212]: CLI server started v1.1.2
2025-06-26 11:19:57,403 [INFO] [MainThread] [cli_server/cli_server.py:104]: Log pipe server started
2025-06-26 11:19:57,664 [DEBUG] [MainThread] [tzlocal/win32.py:55]: Looking up time zone info from registry
2025-06-26 11:19:57,797 [DEBUG] [MainThread] [comtypes/__init__.py:149]: CoInitializeEx(None, 2)
2025-06-26 11:19:58,416 [INFO] [MainThread] [root/bar_manager.py:70]: Starting SystemEventListener...
2025-06-26 11:19:58,416 [INFO] [MainThread] [root/bar_manager.py:70]: Starting KomorebiEventListener...
2025-06-26 11:19:58,416 [INFO] [MainThread] [root/watcher.py:54]: Created file watcher for path C:\Users\DOxenbridge\.config\yasb
2025-06-26 11:19:58,423 [INFO] [Dummy-5] [root/event_listener.py:51]: Created named pipe yasb-acc538a9-522b-11f0-8706-dc97ba7231fc
2025-06-26 11:19:58,454 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:213]: Starting (proactor: <_IocpProactor overlapped#=0 result#=0>)...
2025-06-26 11:19:58,457 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:192]: Thread started
2025-06-26 11:19:58,458 [DEBUG] [MainThread] [systray_widget/systray.py:530]: Loading state from C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:19:58,467 [DEBUG] [TrayMonitor] [systray_widget/systray.py:61]: Systray thread is starting...
2025-06-26 11:19:58,467 [DEBUG] [TaskbarService] [systray_widget/systray.py:75]: TaskbarService thread is starting...
2025-06-26 11:19:58,514 [INFO] [Dummy-5] [root/event_listener.py:125]: Komorebi connected to named pipe: yasb-acc538a9-522b-11f0-8706-dc97ba7231fc
2025-06-26 11:19:58,515 [DEBUG] [TaskbarService] [root/tasks_service.py:75]: Found yasb systray hwnd: 11275990
2025-06-26 11:19:58,515 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 11275990
2025-06-26 11:19:58,547 [DEBUG] [MainThread] [systray_widget/systray.py:278]: Systray thread started
2025-06-26 11:19:58,748 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 11275990
2025-06-26 11:19:58,749 [DEBUG] [TrayMonitor] [systray_widget/tray_monitor.py:208]: Finding real tray hwnd
2025-06-26 11:19:58,758 [DEBUG] [MainThread] [systray_widget/tray_monitor.py:144]: Sending TaskbarCreated message: 49271
2025-06-26 11:19:58,779 [INFO] [MainThread] [root/weather.py:67]: Fetching new weather data at 2025-06-26 11:19:58.779669
2025-06-26 11:19:59,797 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:20:00,791 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:20:00,791 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:20:04,817 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:20:07,846 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:20:07,847 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:20:09,816 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:20:14,821 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:20:19,015 [INFO] [MainThread] [root/controller.py:14]: Reloading Application because of config change.
2025-06-26 11:20:19,127 [INFO] [MainThread] [cli_server/cli_server.py:112]: Log pipe server stopped
2025-06-26 11:20:19,127 [DEBUG] [MainThread] [cli_server/cli_server.py:206]: CLI server stopped
2025-06-26 11:20:19,130 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:20:19,131 [DEBUG] [MainThread] [systray_widget/systray.py:506]: Saving state to disk
2025-06-26 11:20:19,131 [DEBUG] [MainThread] [systray_widget/systray.py:512]: Saving state to C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:20:19,133 [INFO] [MainThread] [root/bar_manager.py:61]: Successfully loaded updated config and re-initialised all bars.
2025-06-26 11:20:19,133 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:218]: Stopping worker thread...
2025-06-26 11:20:19,143 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:201]: Exiting thread
2025-06-26 11:20:19,145 [DEBUG] [MainThread] [qasync._windows._IocpProactor/_windows.py:75]: Closing
2025-06-26 11:20:19,176 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 9373018
2025-06-26 11:20:19,176 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 11275990
2025-06-26 11:20:19,396 [INFO] [MainThread] [root/log.py:59]: YASB v1.7.6
2025-06-26 11:20:19,397 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:212]: CLI server started v1.1.2
2025-06-26 11:20:19,397 [INFO] [MainThread] [cli_server/cli_server.py:104]: Log pipe server started
2025-06-26 11:20:19,539 [DEBUG] [MainThread] [tzlocal/win32.py:55]: Looking up time zone info from registry
2025-06-26 11:20:19,682 [DEBUG] [MainThread] [comtypes/__init__.py:149]: CoInitializeEx(None, 2)
2025-06-26 11:20:20,389 [INFO] [MainThread] [root/bar_manager.py:70]: Starting KomorebiEventListener...
2025-06-26 11:20:20,390 [INFO] [MainThread] [root/bar_manager.py:70]: Starting SystemEventListener...
2025-06-26 11:20:20,390 [INFO] [MainThread] [root/watcher.py:54]: Created file watcher for path C:\Users\DOxenbridge\.config\yasb
2025-06-26 11:20:20,392 [INFO] [Dummy-5] [root/event_listener.py:51]: Created named pipe yasb-b9de4506-522b-11f0-ad9a-dc97ba7231fc
2025-06-26 11:20:20,434 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:213]: Starting (proactor: <_IocpProactor overlapped#=0 result#=0>)...
2025-06-26 11:20:20,436 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:192]: Thread started
2025-06-26 11:20:20,437 [DEBUG] [MainThread] [systray_widget/systray.py:530]: Loading state from C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:20:20,445 [DEBUG] [TrayMonitor] [systray_widget/systray.py:61]: Systray thread is starting...
2025-06-26 11:20:20,446 [DEBUG] [TaskbarService] [systray_widget/systray.py:75]: TaskbarService thread is starting...
2025-06-26 11:20:20,504 [INFO] [Dummy-5] [root/event_listener.py:125]: Komorebi connected to named pipe: yasb-b9de4506-522b-11f0-ad9a-dc97ba7231fc
2025-06-26 11:20:20,512 [DEBUG] [TaskbarService] [root/tasks_service.py:75]: Found yasb systray hwnd: 4132682
2025-06-26 11:20:20,512 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 4132682
2025-06-26 11:20:20,551 [DEBUG] [MainThread] [systray_widget/systray.py:278]: Systray thread started
2025-06-26 11:20:20,713 [INFO] [MainThread] [root/weather.py:67]: Fetching new weather data at 2025-06-26 11:20:20.713275
2025-06-26 11:20:20,752 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 4132682
2025-06-26 11:20:20,753 [DEBUG] [TrayMonitor] [systray_widget/tray_monitor.py:208]: Finding real tray hwnd
2025-06-26 11:20:20,765 [DEBUG] [MainThread] [systray_widget/tray_monitor.py:144]: Sending TaskbarCreated message: 49271
2025-06-26 11:20:21,805 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:20:22,814 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:20:22,814 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:20:26,812 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:20:28,983 [INFO] [MainThread] [root/controller.py:14]: Reloading Application because of config change.
2025-06-26 11:20:29,099 [INFO] [MainThread] [cli_server/cli_server.py:112]: Log pipe server stopped
2025-06-26 11:20:29,100 [DEBUG] [MainThread] [cli_server/cli_server.py:206]: CLI server stopped
2025-06-26 11:20:29,104 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:20:29,105 [DEBUG] [MainThread] [systray_widget/systray.py:506]: Saving state to disk
2025-06-26 11:20:29,105 [DEBUG] [MainThread] [systray_widget/systray.py:512]: Saving state to C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:20:29,107 [INFO] [MainThread] [root/bar_manager.py:61]: Successfully loaded updated config and re-initialised all bars.
2025-06-26 11:20:29,107 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:218]: Stopping worker thread...
2025-06-26 11:20:29,115 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:201]: Exiting thread
2025-06-26 11:20:29,116 [DEBUG] [MainThread] [qasync._windows._IocpProactor/_windows.py:75]: Closing
2025-06-26 11:20:29,151 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 16583628
2025-06-26 11:20:29,151 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 4132682
2025-06-26 11:20:29,313 [INFO] [MainThread] [root/log.py:59]: YASB v1.7.6
2025-06-26 11:20:29,314 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:212]: CLI server started v1.1.2
2025-06-26 11:20:29,314 [INFO] [MainThread] [cli_server/cli_server.py:104]: Log pipe server started
2025-06-26 11:20:29,427 [DEBUG] [MainThread] [tzlocal/win32.py:55]: Looking up time zone info from registry
2025-06-26 11:20:29,549 [DEBUG] [MainThread] [comtypes/__init__.py:149]: CoInitializeEx(None, 2)
2025-06-26 11:20:30,405 [INFO] [MainThread] [root/bar_manager.py:70]: Starting SystemEventListener...
2025-06-26 11:20:30,405 [INFO] [MainThread] [root/bar_manager.py:70]: Starting KomorebiEventListener...
2025-06-26 11:20:30,406 [INFO] [MainThread] [root/watcher.py:54]: Created file watcher for path C:\Users\DOxenbridge\.config\yasb
2025-06-26 11:20:30,421 [INFO] [Dummy-5] [root/event_listener.py:51]: Created named pipe yasb-bfd65f38-522b-11f0-8bd6-dc97ba7231fc
2025-06-26 11:20:30,441 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:213]: Starting (proactor: <_IocpProactor overlapped#=0 result#=0>)...
2025-06-26 11:20:30,445 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:192]: Thread started
2025-06-26 11:20:30,445 [DEBUG] [MainThread] [systray_widget/systray.py:530]: Loading state from C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:20:30,455 [DEBUG] [TrayMonitor] [systray_widget/systray.py:61]: Systray thread is starting...
2025-06-26 11:20:30,455 [DEBUG] [TaskbarService] [systray_widget/systray.py:75]: TaskbarService thread is starting...
2025-06-26 11:20:30,523 [INFO] [Dummy-5] [root/event_listener.py:125]: Komorebi connected to named pipe: yasb-bfd65f38-522b-11f0-8bd6-dc97ba7231fc
2025-06-26 11:20:30,524 [DEBUG] [TaskbarService] [root/tasks_service.py:75]: Found yasb systray hwnd: 25231468
2025-06-26 11:20:30,524 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 25231468
2025-06-26 11:20:30,573 [DEBUG] [MainThread] [systray_widget/systray.py:278]: Systray thread started
2025-06-26 11:20:30,711 [INFO] [MainThread] [root/weather.py:67]: Fetching new weather data at 2025-06-26 11:20:30.711611
2025-06-26 11:20:30,774 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 25231468
2025-06-26 11:20:30,774 [DEBUG] [TrayMonitor] [systray_widget/tray_monitor.py:208]: Finding real tray hwnd
2025-06-26 11:20:30,789 [DEBUG] [MainThread] [systray_widget/tray_monitor.py:144]: Sending TaskbarCreated message: 49271
2025-06-26 11:20:31,835 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:20:32,835 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:20:32,836 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:20:36,849 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:20:39,873 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:20:39,873 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:20:41,327 [INFO] [MainThread] [root/controller.py:14]: Reloading Application because of config change.
2025-06-26 11:20:41,432 [INFO] [MainThread] [cli_server/cli_server.py:112]: Log pipe server stopped
2025-06-26 11:20:41,432 [DEBUG] [MainThread] [cli_server/cli_server.py:206]: CLI server stopped
2025-06-26 11:20:41,437 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:20:41,437 [DEBUG] [MainThread] [systray_widget/systray.py:506]: Saving state to disk
2025-06-26 11:20:41,437 [DEBUG] [MainThread] [systray_widget/systray.py:512]: Saving state to C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:20:41,438 [INFO] [MainThread] [root/bar_manager.py:61]: Successfully loaded updated config and re-initialised all bars.
2025-06-26 11:20:41,439 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:218]: Stopping worker thread...
2025-06-26 11:20:41,448 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:201]: Exiting thread
2025-06-26 11:20:41,450 [DEBUG] [MainThread] [qasync._windows._IocpProactor/_windows.py:75]: Closing
2025-06-26 11:20:41,480 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 10749828
2025-06-26 11:20:41,480 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 25231468
2025-06-26 11:20:41,665 [INFO] [MainThread] [root/log.py:59]: YASB v1.7.6
2025-06-26 11:20:41,667 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:212]: CLI server started v1.1.2
2025-06-26 11:20:41,667 [INFO] [MainThread] [cli_server/cli_server.py:104]: Log pipe server started
2025-06-26 11:20:41,936 [DEBUG] [MainThread] [tzlocal/win32.py:55]: Looking up time zone info from registry
2025-06-26 11:20:42,078 [DEBUG] [MainThread] [comtypes/__init__.py:149]: CoInitializeEx(None, 2)
2025-06-26 11:20:42,734 [INFO] [MainThread] [root/bar_manager.py:70]: Starting KomorebiEventListener...
2025-06-26 11:20:42,735 [INFO] [MainThread] [root/bar_manager.py:70]: Starting SystemEventListener...
2025-06-26 11:20:42,735 [INFO] [MainThread] [root/watcher.py:54]: Created file watcher for path C:\Users\DOxenbridge\.config\yasb
2025-06-26 11:20:42,736 [INFO] [Dummy-5] [root/event_listener.py:51]: Created named pipe yasb-c72fccb0-522b-11f0-9568-dc97ba7231fc
2025-06-26 11:20:42,777 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:213]: Starting (proactor: <_IocpProactor overlapped#=0 result#=0>)...
2025-06-26 11:20:42,782 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:192]: Thread started
2025-06-26 11:20:42,782 [DEBUG] [MainThread] [systray_widget/systray.py:530]: Loading state from C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:20:42,791 [DEBUG] [TaskbarService] [systray_widget/systray.py:75]: TaskbarService thread is starting...
2025-06-26 11:20:42,792 [DEBUG] [TrayMonitor] [systray_widget/systray.py:61]: Systray thread is starting...
2025-06-26 11:20:42,852 [DEBUG] [TaskbarService] [root/tasks_service.py:75]: Found yasb systray hwnd: 9700698
2025-06-26 11:20:42,852 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 9700698
2025-06-26 11:20:42,857 [INFO] [Dummy-5] [root/event_listener.py:125]: Komorebi connected to named pipe: yasb-c72fccb0-522b-11f0-9568-dc97ba7231fc
2025-06-26 11:20:42,890 [DEBUG] [MainThread] [systray_widget/systray.py:278]: Systray thread started
2025-06-26 11:20:43,092 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 9700698
2025-06-26 11:20:43,092 [DEBUG] [TrayMonitor] [systray_widget/tray_monitor.py:208]: Finding real tray hwnd
2025-06-26 11:20:43,103 [DEBUG] [MainThread] [systray_widget/tray_monitor.py:144]: Sending TaskbarCreated message: 49271
2025-06-26 11:20:43,125 [INFO] [MainThread] [root/weather.py:67]: Fetching new weather data at 2025-06-26 11:20:43.125192
2025-06-26 11:20:44,142 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:20:45,137 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:20:45,138 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:20:49,158 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:20:52,201 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:20:52,201 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:20:54,163 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:20:56,459 [INFO] [MainThread] [root/controller.py:14]: Reloading Application because of config change.
2025-06-26 11:20:56,559 [INFO] [MainThread] [cli_server/cli_server.py:112]: Log pipe server stopped
2025-06-26 11:20:56,560 [DEBUG] [MainThread] [cli_server/cli_server.py:206]: CLI server stopped
2025-06-26 11:20:56,563 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:20:56,563 [DEBUG] [MainThread] [systray_widget/systray.py:506]: Saving state to disk
2025-06-26 11:20:56,563 [DEBUG] [MainThread] [systray_widget/systray.py:512]: Saving state to C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:20:56,565 [INFO] [MainThread] [root/bar_manager.py:61]: Successfully loaded updated config and re-initialised all bars.
2025-06-26 11:20:56,565 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:218]: Stopping worker thread...
2025-06-26 11:20:56,574 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:201]: Exiting thread
2025-06-26 11:20:56,575 [DEBUG] [MainThread] [qasync._windows._IocpProactor/_windows.py:75]: Closing
2025-06-26 11:20:56,612 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 8651142
2025-06-26 11:20:56,612 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 9700698
2025-06-26 11:20:56,782 [INFO] [MainThread] [root/log.py:59]: YASB v1.7.6
2025-06-26 11:20:56,783 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:212]: CLI server started v1.1.2
2025-06-26 11:20:56,783 [INFO] [MainThread] [cli_server/cli_server.py:104]: Log pipe server started
2025-06-26 11:20:56,902 [DEBUG] [MainThread] [tzlocal/win32.py:55]: Looking up time zone info from registry
2025-06-26 11:20:57,031 [DEBUG] [MainThread] [comtypes/__init__.py:149]: CoInitializeEx(None, 2)
2025-06-26 11:20:57,954 [INFO] [MainThread] [root/bar_manager.py:70]: Starting SystemEventListener...
2025-06-26 11:20:57,955 [INFO] [MainThread] [root/bar_manager.py:70]: Starting KomorebiEventListener...
2025-06-26 11:20:57,955 [INFO] [MainThread] [root/watcher.py:54]: Created file watcher for path C:\Users\DOxenbridge\.config\yasb
2025-06-26 11:20:57,975 [INFO] [Dummy-5] [root/event_listener.py:51]: Created named pipe yasb-d042406e-522b-11f0-aeaa-dc97ba7231fc
2025-06-26 11:20:58,002 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:213]: Starting (proactor: <_IocpProactor overlapped#=0 result#=0>)...
2025-06-26 11:20:58,003 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:192]: Thread started
2025-06-26 11:20:58,003 [DEBUG] [MainThread] [systray_widget/systray.py:530]: Loading state from C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:20:58,012 [DEBUG] [TrayMonitor] [systray_widget/systray.py:61]: Systray thread is starting...
2025-06-26 11:20:58,013 [DEBUG] [TaskbarService] [systray_widget/systray.py:75]: TaskbarService thread is starting...
2025-06-26 11:20:58,079 [INFO] [Dummy-5] [root/event_listener.py:125]: Komorebi connected to named pipe: yasb-d042406e-522b-11f0-aeaa-dc97ba7231fc
2025-06-26 11:20:58,081 [DEBUG] [TaskbarService] [root/tasks_service.py:75]: Found yasb systray hwnd: 6557632
2025-06-26 11:20:58,081 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 6557632
2025-06-26 11:20:58,115 [DEBUG] [MainThread] [systray_widget/systray.py:278]: Systray thread started
2025-06-26 11:20:58,268 [INFO] [MainThread] [root/weather.py:67]: Fetching new weather data at 2025-06-26 11:20:58.268366
2025-06-26 11:20:58,315 [DEBUG] [TaskbarService] [systray_widget/tasks_service.py:81]: Adding TaskbandHWND prop to hwnd 6557632
2025-06-26 11:20:58,315 [DEBUG] [TrayMonitor] [systray_widget/tray_monitor.py:208]: Finding real tray hwnd
2025-06-26 11:20:58,328 [DEBUG] [MainThread] [systray_widget/tray_monitor.py:144]: Sending TaskbarCreated message: 49271
2025-06-26 11:20:59,420 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:20:59,421 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:20:59,452 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:21:00,492 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:21:00,492 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:21:04,437 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:21:07,400 [DEBUG] [MainThread] [systray_widget/systray.py:466]: Re-sorting widgets
2025-06-26 11:21:07,400 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:21:09,437 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:21:14,446 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:21:19,441 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:21:24,447 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:21:29,449 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:21:34,444 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:21:39,446 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:21:44,441 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:21:46,336 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:253]: CLI server received command: toggle-bar
2025-06-26 11:21:49,439 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:21:54,436 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:21:56,280 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:253]: CLI server received command: hide-bar
2025-06-26 11:21:59,448 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:22:03,048 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:253]: CLI server received command: toggle-bar
2025-06-26 11:22:04,450 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:22:09,446 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:22:14,441 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:22:19,446 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:22:24,444 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:22:29,442 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:22:34,450 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:22:39,441 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:22:44,442 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:22:49,451 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:22:54,451 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:22:59,449 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:23:04,442 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:23:09,442 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:23:14,446 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:23:19,450 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:23:24,451 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:23:29,440 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:23:34,449 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:23:39,447 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:23:44,444 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:23:49,452 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:23:54,453 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:23:59,443 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:24:04,455 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:24:09,441 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:24:14,444 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:24:19,447 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:24:24,447 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:24:29,455 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:24:34,447 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:24:39,455 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:24:44,452 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:24:49,453 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:24:54,448 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:24:59,446 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:25:04,448 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:25:09,447 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:25:14,451 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:25:19,448 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:25:24,448 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:25:29,444 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:25:34,447 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:25:39,443 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:25:44,441 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:25:49,453 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:25:54,445 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:25:59,440 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:26:04,444 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:26:09,447 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:26:14,443 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:26:19,446 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:26:24,443 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:26:29,446 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:26:34,444 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:26:39,451 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:26:44,444 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:26:49,447 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:26:54,447 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:26:59,444 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:27:04,457 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:27:09,445 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:27:14,448 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:27:19,449 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:27:24,438 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:27:29,453 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:27:34,443 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:27:39,445 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:27:44,451 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:27:49,444 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:27:54,440 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:27:59,444 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:28:04,440 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:28:09,442 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:28:14,445 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:28:19,439 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:28:24,452 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:28:29,455 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:28:34,449 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:28:38,468 [WARNING] [Dummy-5] [root/event_listener.py:82]: Pipe has been ended: (109, 'PeekNamedPipe', 'The pipe has been ended.')
2025-06-26 11:28:38,468 [INFO] [Dummy-5] [root/event_listener.py:92]: Attempting to reconnect to Komorebi...
2025-06-26 11:28:39,454 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:28:41,469 [INFO] [Dummy-5] [root/event_listener.py:51]: Created named pipe yasb-d042406e-522b-11f0-aeaa-dc97ba7231fc
2025-06-26 11:28:41,518 [INFO] [Dummy-5] [root/event_listener.py:125]: Komorebi connected to named pipe: yasb-d042406e-522b-11f0-aeaa-dc97ba7231fc
2025-06-26 11:28:44,445 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:28:49,442 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:28:54,451 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:28:57,011 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:253]: CLI server received command: toggle-bar
2025-06-26 11:28:58,427 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:253]: CLI server received command: toggle-bar
2025-06-26 11:28:59,454 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:29:04,454 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:29:09,452 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:29:14,450 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:29:19,445 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:29:24,459 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:29:29,442 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:29:34,441 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:29:39,440 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:29:44,450 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:29:49,442 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:29:54,442 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:29:59,445 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:30:04,453 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:30:09,445 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:30:14,449 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:30:19,454 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:30:24,450 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:30:29,455 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:30:34,457 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:30:39,446 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:30:44,444 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:30:49,450 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:30:54,443 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:30:57,440 [INFO] [MainThread] [root/weather.py:67]: Fetching new weather data at 2025-06-26 11:30:57.440820
2025-06-26 11:30:59,452 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:31:04,443 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:31:09,449 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:31:14,444 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:31:19,442 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:31:24,447 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:31:29,455 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:31:34,457 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:31:39,445 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:31:39,989 [WARNING] [Dummy-5] [root/event_listener.py:82]: Pipe has been ended: (109, 'PeekNamedPipe', 'The pipe has been ended.')
2025-06-26 11:31:39,990 [INFO] [Dummy-5] [root/event_listener.py:92]: Attempting to reconnect to Komorebi...
2025-06-26 11:31:42,990 [INFO] [Dummy-5] [root/event_listener.py:51]: Created named pipe yasb-d042406e-522b-11f0-aeaa-dc97ba7231fc
2025-06-26 11:31:43,052 [INFO] [Dummy-5] [root/event_listener.py:125]: Komorebi connected to named pipe: yasb-d042406e-522b-11f0-aeaa-dc97ba7231fc
2025-06-26 11:31:44,455 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:31:49,447 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:31:54,452 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:31:59,455 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:32:04,442 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:32:09,444 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:32:14,459 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:32:19,452 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:32:24,447 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:32:29,447 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:32:34,445 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:32:39,449 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:32:44,450 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:32:49,456 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:32:54,450 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:32:59,458 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:33:04,453 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:33:09,455 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:33:14,442 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:33:19,455 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:33:24,452 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:33:29,448 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:33:34,448 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:33:39,449 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:33:44,452 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:33:49,452 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:33:54,451 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:33:59,454 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:34:04,446 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:34:09,447 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:34:14,445 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:34:19,448 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:34:24,451 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:34:29,446 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:34:34,454 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:34:39,450 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:34:44,453 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:34:49,455 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:34:54,452 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:34:59,457 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:35:04,455 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:35:09,462 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:35:14,446 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:35:19,449 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:35:24,457 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:35:29,444 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:35:34,450 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:35:39,462 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:35:44,451 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:35:49,450 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:35:54,453 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:35:59,448 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:36:04,452 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:36:09,445 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:36:14,451 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:36:19,446 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:36:24,459 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:36:29,452 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:36:34,452 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:36:39,452 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:36:44,452 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:36:49,452 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:36:54,449 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:36:59,445 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:37:04,456 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:37:09,445 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:37:14,450 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:37:19,454 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:37:24,458 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:37:29,461 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:37:34,444 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:37:39,454 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:37:44,459 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:37:49,453 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:37:54,463 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:37:59,451 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:38:04,453 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:38:09,453 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:38:14,461 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:38:19,451 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:38:24,448 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:38:29,450 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:38:34,475 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:38:39,449 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:38:44,450 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:38:49,461 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:38:54,451 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:38:59,448 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:39:04,460 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:39:09,446 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:39:14,445 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:39:19,453 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:39:24,452 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:39:29,445 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:39:34,458 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:39:39,459 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:39:44,448 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:39:49,445 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:39:54,453 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:39:59,446 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:40:04,462 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:40:09,455 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:40:14,458 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:40:19,457 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:40:24,457 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:40:29,454 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:40:34,468 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:40:39,457 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:40:44,455 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:40:49,462 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:40:54,458 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:40:57,452 [INFO] [MainThread] [root/weather.py:67]: Fetching new weather data at 2025-06-26 11:40:57.452203
2025-06-26 11:40:59,456 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:41:04,450 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:41:09,462 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:41:14,460 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:41:19,448 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:41:24,460 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:41:29,461 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:41:34,449 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:41:39,452 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:41:44,456 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:41:49,468 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:41:54,460 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:41:59,448 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:42:04,452 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:42:09,447 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:42:14,453 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:42:19,450 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:42:24,455 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:42:29,455 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:42:34,452 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:42:39,512 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:42:44,468 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:42:49,456 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:42:54,457 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:42:59,453 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:43:04,460 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:43:09,454 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:43:14,448 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:43:19,460 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:43:24,451 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:43:29,455 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:43:34,451 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:43:39,454 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:43:44,456 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:43:49,463 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:43:54,457 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:43:59,459 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:44:04,462 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:44:09,454 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:44:14,455 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:44:19,453 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:44:24,452 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:44:29,450 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:44:34,459 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:44:39,460 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:44:44,456 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:44:49,454 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:44:54,454 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:44:59,455 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:45:04,453 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:45:09,448 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:45:14,461 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:45:19,466 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:45:24,461 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:45:29,450 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:45:34,450 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:45:39,455 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:45:44,462 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:45:49,453 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:45:53,994 [WARNING] [Dummy-5] [root/event_listener.py:82]: Pipe has been ended: (109, 'PeekNamedPipe', 'The pipe has been ended.')
2025-06-26 11:45:53,994 [INFO] [Dummy-5] [root/event_listener.py:92]: Attempting to reconnect to Komorebi...
2025-06-26 11:45:54,452 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:45:56,994 [INFO] [Dummy-5] [root/event_listener.py:51]: Created named pipe yasb-d042406e-522b-11f0-aeaa-dc97ba7231fc
2025-06-26 11:45:59,457 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:46:04,461 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:46:09,451 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:46:14,154 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:253]: CLI server received command: toggle-bar
2025-06-26 11:46:14,459 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:46:19,453 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:46:22,335 [INFO] [Dummy-5] [root/event_listener.py:125]: Komorebi connected to named pipe: yasb-d042406e-522b-11f0-aeaa-dc97ba7231fc
2025-06-26 11:46:24,455 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:46:29,454 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:46:34,455 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:46:35,153 [INFO] [Thread-1 (_run_server)] [cli_server/cli_server.py:253]: CLI server received command: toggle-bar
2025-06-26 11:46:39,454 [DEBUG] [TrayMonitor] [systray_widget/utils.py:124]: Could not open process ID 6152. Err: 5
2025-06-26 11:46:39,710 [INFO] [MainThread] [root/controller.py:29]: Exiting Application from tray...
2025-06-26 11:46:39,822 [INFO] [MainThread] [cli_server/cli_server.py:112]: Log pipe server stopped
2025-06-26 11:46:39,822 [DEBUG] [MainThread] [cli_server/cli_server.py:206]: CLI server stopped
2025-06-26 11:46:39,823 [DEBUG] [MainThread] [systray_widget/systray.py:488]: Updating current state
2025-06-26 11:46:39,823 [DEBUG] [MainThread] [systray_widget/systray.py:506]: Saving state to disk
2025-06-26 11:46:39,823 [DEBUG] [MainThread] [systray_widget/systray.py:512]: Saving state to C:\Users\DOxenbridge\AppData\Local\Yasb\systray_state_LGELECTRONICSLGULTRAWIDE407NTBK99581.json
2025-06-26 11:46:39,824 [DEBUG] [MainThread] [qasync._windows._EventPoller/_windows.py:218]: Stopping worker thread...
2025-06-26 11:46:39,838 [DEBUG] [Dummy-8] [qasync._windows._EventWorker/_windows.py:201]: Exiting thread
2025-06-26 11:46:39,840 [DEBUG] [MainThread] [qasync._windows._IocpProactor/_windows.py:75]: Closing
2025-06-26 11:46:39,856 [ERROR] [Dummy-34] [root/event_service.py:32]: Failed to emit signal <bound PYQT_SIGNAL window_name_change of ActiveWindowWidget object at 0x0000020F43B3BD90>. Removing link to WinEvent.EventObjectNameChange.
2025-06-26 11:46:39,877 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 25362540
2025-06-26 11:46:39,877 [DEBUG] [MainThread] [systray_widget/utils.py:96]: Destroying window 6557632
